README.md

```diff
@@ -117,7 +117,9 @@ meta.nonEventListenerTestCompliant = true;
 ```
 
 ## CLI Tools
+We have created a package that contains all of the commands to run our tools. If you only want to use some of our packages, you can install each package you want and it comes with its own command.
```

Comment:
I think we want to direct people to consume the packages separately and deprecate the `scripts` repo soon. So we don't have to put in this paragraph